Wikipedia/Code/WMFRelatedSectionBlackList.m

```diff
@@ -51,70 +60,20 @@ + (instancetype)loadFromDisk {
 
 - (void)addBlackListTitle:(MWKTitle*)title {
```

Comment:
I moved the code into `addEntry:`, et al. But as far as the locking itself - I wanted to leave it in there because it is inconsequential. Since this class only gets one thing added on occasion, the locking is fairly cheap.